.azure/asgardeo-java-oidc-sdk-sca-scan.yaml

```diff
@@ -45,3 +45,10 @@ jobs:
         parameters:
           API_KEY: $(FOSSA-API-KEY)
           BRANCH: master
+      - template: ci-pipelines/templates/sca-scan-jfrog.yaml@templates
+        parameters:
+          PROJECT_TYPE: mvn
+          GITHUB_CONNECTION: $(GITHUB_CONNECTION) # GitHub connection name to Show the vulnerability report as a PR comment
+          ACCESS_TOKEN: $(ACCESS_TOKEN) # JFrog access token
+          SERVER_ID: $(CONNECTION_NAME) # JFrog SERVER ID
```

Comment:
Move this and the following parameters to the template.

Reply:
Done, only Access token has to be provided as a parameter